src/BaseCarousel/glide/glide.setup.js

```diff
@@ -127,15 +127,20 @@ function setupCaroarousel({
     // Set .slider-start for starting gradient styles.
     const gliderElement = document.querySelector(`#${instanceName}`);
     if (!gliderElement) return; // necessary. it breaks on resize
-    gliderElement.classList.add("slider-start");
+    const gliderTrack = gliderElement.querySelector('.glide__track');
+
+    gliderTrack.classList.add("slider-start");
   });
 
   // On Move event...
   slider.on("move", () => {
     // Get glider top level element.
     const gliderElement = document.querySelector(`#${instanceName}`);
+    
     if (!gliderElement) return; // necessary. it breaks on resize
+    const gliderTrack = gliderElement.querySelector('.glide__track');
 
+    console.log({gliderElement});
```

Comment:
Maybe this should be removed

Reply:
hah, thanks I didn't notice, removed now.